Dependencies/xr/Include/XR.h

```diff
@@ -177,6 +177,7 @@ namespace xr
         static constexpr float DEFAULT_DEPTH_NEAR_Z{ 0.5f };
         static constexpr float DEFAULT_DEPTH_FAR_Z{ 1000.f };
         static constexpr uint32_t DEFAULT_CONTROLLER_BUTTONS{ 4 };
```

Comment:
>DEFAULT_CONTROLLER_BUTTONS [](http://example.com/codeflow?start=34&length=26)

nit: can we get these changed to DEFAULT_CONTROLLER_BUTTON_COUNT etc so that folks know this is a length. right now it reads like the value is a button object

Reply:
or NUM_DEFAULT_CONTROLLER_BUTTONS I don't personally have a strong preference on how the fact this is a length/count is expressed